src/COCOA/wwwroot/js/components/LoginPage.jsx

```diff
@@ -32,7 +32,7 @@ class LoginPage extends React.Component {
         var email = this.state.email;
         var password = this.state.password;
 
-        xhr.open('get', "/user/signin?email=" + email + "&password=" + password + "&persistent=true", true);
+        xhr.open('get', "/user/signin?email=" + encodeURIComponent(email) + "&password=" + encodeURIComponent(password) + "&persistent=true", true);
```

Comment:
Add checkbox on loginpage: "Keep me signed in".
Cast bool to string: "true/false".